source/nodejs/adaptivecards-site/package.json

```diff
@@ -5,6 +5,7 @@
 	"scripts": {
 		"clean": "hexo clean",
 		"build": "hexo generate",
+		"release-deps": "npx lerna run --scope adaptivecards-site --include-dependencies release --stream",
```

Comment:
>"release-deps": "npx lerna run --scope adaptivecards-site --include-dependencies release --stream", [](http://example.com/codeflow?start=2&length=99)

will this require pipeline changes?

Reply:
no - this is just a little helper script... you can `npm run release-deps` in `source/nodejs/adaptivecards-site`, and it causes `lerna` to build only what's needed for the site. I just got tired of typing that command :)